index.js

```diff
@@ -104,3 +104,16 @@ function openModal(e) {
 modalBtns.forEach((modalBtn) => {
   modalBtn.addEventListener('click', openModal);
 });
+
+const regularExpression = /^[_a-z0-9-]+(\.[_a-z0-9-]+)*(\+[a-z0-9-]+)?@[a-z0-9-]+(\.[a-z0-9-]+)*$/;
+const emailField = document.querySelector('#mail');
+const getInTouch = document.querySelector('.button-submit');
+
+getInTouch.addEventListener('click', () => {
+  const validatedMail = emailField.value;
+  if (!regularExpression.test(validatedMail)) {
+    emailField.setCustomValidity('Email should be in lowercase. Please input the email in lowercase!');
+  } else {
+    emailField.setCustomValidity('');
+  }
+});
```

Comment:
- [ ] Good job adding the validations for the form, however, the email error message should show up close to the "submit" button. Kindly make this change.